code/modules/clothing/under/skirt_dress.dm

```diff
@@ -99,3 +99,13 @@
 	icon_state = "plaid_green"
 	item_state = "plaid_green"
 	item_color = "plaid_green"
+
+/obj/item/clothing/under/dress/skirt/summer
+	desc = "A revealing outfit perfect for a warm summer day."
+	custom_price = 75
```

Comment:
what is this bogus type with no icon state and why use a custom price if you're going to use the standard vendor price anyway